crates/biome_css_formatter/tests/specs/css/pseudo/pseudo_class_function_compound_selector_list.css.snap

```diff
@@ -0,0 +1,46 @@
+---
+source: crates/biome_formatter_test/src/snapshot_builder.rs
+info: css/pseudo/pseudo_class_function_compound_selector_list.css
+---
+
+# Input
+
+```css
+:-webkit-any(i,p,:link,span:focus) {}
+:-webkit-any(
+    i,  p  
+    
+    , :link  ,
+
+span:focus
+) {}
+```
+
+
+=============================
+
+# Outputs
+
+## Output 1
+
+-----
+Indent style: Tab
+Indent width: 2
+Line ending: LF
+Line width: 80
+-----
+
+```css
+:-webkit-any(i, p, :link, span:focus) {
+}
+:-webkit-any(
+		i,
+		p,
+		:link,
+
+		span:focus
+	) {
+}
```

Comment:
I'm not sure but prettier formats them the same way:
https://prettier.io/playground/#N4Igxg9gdgLgprEAuZBaA7nARgawJYyoCGUAngBR4A0ADlUgDZ5Q5UDONJSAZhGAK5sAlAAJgAXwA6UJBmz5CJCtJGqR1VTVXSVaqiMbMcqqjqgcuvAW2miJIKiAg0YeaG2SgiAJ28R0AAo+CB4oRAzoRKQejljeRGA4cDAAykQAtnAAMsxwyNzhbHCx8YnJKZxgzADmyDDe-MUgRel4dQ1NcAAeNHDeeJmw4QAqfVA+eHChBQxFjmw1DHAAivwQ8PmFTQBWbF0piytrGzxbjgCOx3ABfjShIERsqFBwcAAm7w4g9UR4TFDVADCEHS6SIyAeDAYXwWAKWAEEYPU8Fh+PAAn0ci9NrMmgALGDpBgAdTxBCmlTgKRCBDwADcCKQIWA2DEQHTGgBJKAfWApMD9FzwnkpGCkJY4uYgGh+IrE+I0CEyqZ9Ol5RzMIreGA3IjVMGSpqcbxa5msr4y5gwYl4N4wPHIAAcAAZHN44Jc8O7dfrwadcY4YEQsDa7Q6kAAmRyCODDYPTM4gODpLDvD5vLIkar8PVwABiEG8YKRNQhRDREBA4nEQA

Reply:
This is the result of using the `join_with_*` builders right now, because they try to respect the newlines from the input source. It comes up in a few places where we want to just ignore the input and always output single newlines (like selector lists) or keep them flat (like this case).

I think I'll do a pass on all of these and fix them in one go in a follow up PR today.